Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelSaveTests.swift

```diff
@@ -6,11 +6,13 @@ final class SitterProfileViewModelSaveTests: XCTestCase {
 
     override func setUp() {
         super.setUp()
+        KeyValueStorage.clearStoredSitterData()
         URLProtocol.registerClass(MockURLProtocol.self)
         viewModel = SitterProfileViewModel()
     }
 
     override func tearDown() {
+        KeyValueStorage.clearStoredSitterData()
```

Comment:
Why do we need to call this function in both `setUp()` and `tearDown()`?

Reply:
it's not obligatory, deleted `KeyValueStorage.clearStoredSitterData()` for setUp() with 34c6879